examples/qunit_simple/tests.js

```diff
@@ -1,8 +1,8 @@
-test('says hello world', function(){
+QUnit.test('says hello world', function(){
```

Comment:
this change is not necessary yet as the global methods like `test`, `module`, etc will still be supported until QUnit 2.0.0.

Even though, if you like them namespaced, as I, I would suggest to use the assert argument, example:

```js
QUnit.test('says hello world', function(assert){
    assert.equal(hello(), 'hello world', 'should equal hello world');
```